docs/mms/index.md

```diff
@@ -0,0 +1,107 @@
+* [Export an ONNX Model to Serve with MMS](https://github.com/awslabs/mxnet-model-server/docs/export_from_onnx.md)
+
+## Getting Started
+
+To install MMS with ONNX support, make sure you have Python installed, then for Ubuntu run:
+
+```bash
+sudo apt-get install protobuf-compiler libprotoc-dev
+pip install mxnet-model-server
+```
+
+Or for Mac run:
+
+```bash
+conda install -c conda-forge protobuf
+pip install mxnet-model-server
+```
+
+
+## Serving a Model
+
+To serve a model you must first create or download a model archive. Visit the [model zoo](https://github.com/awslabs/mxnet-model-server/docs/model_zoo.md) to browse the free models. MMS options can be explored as follows:
+
+```bash
+mxnet-model-server --help
+```
+
+Here is an easy example for serving an object classification model. You can use any URI and the model will be downloaded first, then served from that location:
+
+```bash
+mxnet-model-server \
+  --models squeezenet=https://s3.amazonaws.com/model-server/models/squeezenet_v1.1/squeezenet_v1.1.model
+```
+
+
+### Test Inference on a Model
+
+Assuming you have run the previous `mxnet-model-server` command to start serving the object classification model, you can now upload an image to its `predict` REST API endpoint. The following will download a picture of a kitten, then upload it to the prediction endpoint.
+
+```bash
+curl -O https://s3.amazonaws.com/model-server/inputs/kitten.jpg
+curl -X POST http://127.0.0.1:8080/squeezenet/predict -F "data=@kitten.jpg"
+```
+
+The predict endpoint will return a prediction response in JSON. It will look something like the following result:
+
+```
+{
+  "prediction": [
+    [
+      {
+        "class": "n02124075 Egyptian cat",
+        "probability": 0.9408261179924011
+      },
+...
+```
+
+For more examples of serving models visit the following resources:
+
+* [Quickstart: Model Serving](https://github.com/awslabs/mxnet-model-server/README.md#serve-a-model)
+* [Running the Model Server](https://github.com/awslabs/mxnet-model-server/docs/server.md)
+
+
+## Create a Model Archive
+
+Creating a model archive involves rounding up the required model artifacts, then using the `mxnet-model-export` command line interface. As the process for creating archives is likely to evolve as the project adds features it is recommended that you review the following resources to get the latest instructions:
+
+* [Quickstart: Export a Model](https://github.com/awslabs/mxnet-model-server/README.md#export-a-model)
+* [Model Artifacts](https://github.com/awslabs/mxnet-model-server/docs/export_model_file_tour.md)
+* [Loading and Serving Gluon Models](https://github.com/awslabs/mxnet-model-server/tree/master/examples/gluon_alexnet)
+* [Creating a MMS Model Archive from an ONNX Model](https://github.com/awslabs/mxnet-model-server/docs/export_from_onnx.md)
+* [Create an ONNX model (that will run with MMS) from PyTorch](https://github.com/onnx/onnx-mxnet/blob/master/README.md#quick-start)
+
+
+## Using Containers
```

Comment:
Should we point users to the Docker Hub builds?
https://hub.docker.com/r/awsdeeplearningteam/mms_cpu and https://hub.docker.com/r/awsdeeplearningteam/mms_gpu

Reply:
I added a docker pull command and a link to the parent on docker hub. I think it would be nicer to have an actual organization page and have the repos listed there. One stop shopping, but this should be fine for now!